packages/tools/src/isObject.ts

```diff
@@ -0,0 +1,4 @@
+export function isObject(obj: unknown): obj is object {
```

Comment:
pls deprecate the old function from `apps/meteor/lib/utils/isObject.ts` mentioning to use this one